ember/app/cases/detail/route.js

```diff
@@ -1,7 +1,13 @@
 import Route from "@ember/routing/route";
-
+import { useCalumaQuery } from "@projectcaluma/ember-core/caluma-query";
+import { allCases } from "@projectcaluma/ember-core/caluma-query/queries";
 export default class CasesDetailRoute extends Route {
-  model(params) {
-    return params.id;
+  model({ case_id }) {
+    const caseQuery = useCalumaQuery(this, allCases, () => ({
+      options: { pageSize: 1 },
+      filter: [{ id: case_id }],
+    }));
```

Comment:
This is not how `useCalumaQuery` works. This needs to be a property on the controller.. Please check out the documentation in `ember-caluma`

Reply:
Instead of fetching the same case/work item (duplicating this code snippet) in the child controllers, i opted to fetch them in the parent route model. What would be the best alternative to `useCalumaQuery`, just using apollo?

Reply:
Changed to apollo queries in controllers for both cases